ext/date/tests/date_constants.phpt

```diff
@@ -37,6 +37,38 @@ Date constants
         DATE_RSS     == DateTime::RSS,
         DATE_W3C     == DateTime::W3C
     );
+
+    print "\n";
+
+    var_dump(
```

Comment:
Instead of adding to this test case, I think the additions should go into a new test case (or test cases).